src/app/account-details/personal-details.component.ts

```diff
@@ -158,10 +160,20 @@ export class PersonalDetailsComponent {
             .post('/rest/v1/account/details', updates)
             .pipe(map((res: HttpResponse<any>) => res['result']))
```

Comment:
Do we want a catchError and show the "Failed ..." message when the http throws?

Reply:
we have an http interceptor service (in rmmapi/) which catches these (I think everywhere)